src/DraftEditor.tsx

```diff
@@ -511,7 +511,7 @@ class DraftEditor extends Component<IDraftEditorProps, IDraftEditorState> {
     onTab = (e: React.KeyboardEvent<{}>) => {
         const { editorState, valueSearchOpen, searchString } = this.state;
         if (valueSearchOpen) {
-            if (e.key === 'Tab') {
+            if (e.key === 'Tab' && document.querySelector('.value-mention-item-focused')) {
```

Comment:
assign document.querySelector('.value-mention-item-focused') to a variable and use

Reply:
ok bro